src/main/java/com/pedjak/gradle/plugins/dockerizedtest/DockerizedExecHandle.java

```diff
@@ -252,16 +279,13 @@ public ExecHandle start() {
             }
             setState(ExecHandleState.STARTING);
 
-            execHandleRunner = new DockerizedExecHandleRunner(this, streamsHandler, executorFactory);
-            executor.execute(new BuildOperationIdentifierPreservingRunnable(execHandleRunner));
+            execHandleRunner = new DockerizedExecHandleRunner(this, new CompositeStreamsHandler(), executor);
+            executor.execute(new CurrentBuildOperationPreservingRunnable(execHandleRunner));
 
             while (stateIn(ExecHandleState.STARTING)) {
                 LOGGER.debug("Waiting until process started: {}.", displayName);
                 try {
-                    if (!stateChanged.await(30, TimeUnit.SECONDS)) {
```

Comment:
Your PR has removed the logic introduced in https://github.com/pedjak/gradle-dockerized-test-plugin/commit/69073c3dec2b300fb6a6c0f03f828ce599ca6280